7_singularity_containers/bootstrap.def

```diff
@@ -0,0 +1,25 @@
+Bootstrap: docker
+From: ubuntu:latest
+
+%runscript
+
+exec echo "The runscript is the containers default runtime command!"
+
+
+%files
+/home/tcooper/bootstrap.def /data/bootstrap.def
```

Comment:
I cannot access that file, anyway I think that it would be better to include that file in the repo, so that also people with no comet access can take a look at it